apps/mull-ui/src/app/components/contact-row/contact-row.scss

```diff
@@ -16,6 +16,7 @@
     display: flex;
     padding: 0.3rem;
     flex-direction: column;
+    align-self: center;
```

Comment:
💡 Seems like everything in this flex is meant to be centered. You could remove this and add `align-items: center;` to `.contact-container`

Reply:
The reason why I put `align-self: center` is to center the name in respect to the profile picture.
![image](https://user-images.githubusercontent.com/38992971/113467556-9af47900-9411-11eb-8af8-83a64bc854ec.png)

Compared to `align-items: center` in `.contact-container`
![image](https://user-images.githubusercontent.com/38992971/113467536-84e6b880-9411-11eb-8a74-388de044be0c.png)